app/redux/reducers/SetsReducer.js

```diff
@@ -171,9 +171,25 @@ const saveWorkoutSet = (state, action) => {
 // DELETE_WORKOUT_SET
 
 const deleteWorkoutSet = (state, action) => {
+    // copy workout data
+    let newWorkoutData = state.workoutData.slice(0);
+
+    // get set
+    let setIndex = newWorkoutData.findIndex( set => set.setID === action.setID );
+
+    newWorkoutData[setIndex].exercise = "";
+    newWorkoutData[setIndex].weight = "";
+    newWorkoutData[setIndex].rpe = "";
+    newWorkoutData[setIndex].tags = [];
```

Comment:
- [ ] we discussed this, while i wanted it to be done this way, it was decided that we actually need the data to be kept and for the set to be restored to its original state